core/src/utils/stores.ts

```diff
@@ -141,6 +141,13 @@ export const toReadableStore = <T>(x: ReadableSignal<T> | T) => (isStore(x) ? x
  */
 export const toWritableStore = <T>(x: WritableSignal<T> | T) => (isStore(x) ? x : writable(x));
 
+/**
+ *  If the provided argument is a store, its value is returned, otherwise, the value is returned
```

Comment:
```suggestion
 * If the provided argument is a store, its value is returned, otherwise, the value is returned
```
(there seems to be an extra space before `If the provided argument`)